src/WireMock.Net.OpenApiParser/Utils/ExampleValueGenerator.cs

```diff
@@ -28,38 +29,55 @@ public object GetExampleValue(OpenApiSchema schema)
             switch (schema?.GetSchemaType())
             {
                 case SchemaType.Boolean:
-                    return _settings.ExampleValues.Boolean;
+                    OpenApiBoolean exampleBoolean = (OpenApiBoolean)schema?.Example;
```

Comment:
You can just use `var` (multiple places in this file)

Reply:
I changed the declaration type to var, the thing here is that `schema?.Example` is type `IOpenApiAny` then I need to force the cast to access the property `Value`.